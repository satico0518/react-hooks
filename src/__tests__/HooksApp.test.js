import React from 'react'

const { shallow } = require('enzyme');
const { default: HooksApp } = require('../HooksApp');

describe('HooksApp', () => {
  test('should render ok', () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
