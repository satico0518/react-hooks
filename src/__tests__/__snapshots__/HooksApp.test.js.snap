// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HooksApp should render ok 1`] = `
<div>
  hooks
</div>
`;
